Examples/minimum_user_for_s2e/src/src_user/Test/test/src_core/TlmCmd/test_common_cmd_packet_util.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+
+import isslwings as wings
+import pytest
+
+ROOT_PATH = "../../../"
+sys.path.append(os.path.dirname(__file__) + "/" + ROOT_PATH + "utils")
+import c2a_enum_utils
+import wings_utils
+
+c2a_enum = c2a_enum_utils.get_c2a_enum()
+ope = wings_utils.get_wings_operation()
+
+
+@pytest.mark.real
+@pytest.mark.sils
+def test_ccp_register_tlc_asap():
+    # 登録前に TL_GS をクリア
+    clear_tl_gs()
+
+    # GS にセット
+    assert "SUC" == wings.util.send_rt_cmd_and_confirm(
+        ope,
+        c2a_enum.Cmd_CODE_TLCD_SET_ID_FOR_TLM,
+        (c2a_enum.TLCD_ID_FROM_GS,),
+        c2a_enum.Tlm_CODE_HK,
+    )
+
+    tlm_TL = wings.util.generate_and_receive_tlm(
+        ope, c2a_enum.Cmd_CODE_GENERATE_TLM, c2a_enum.Tlm_CODE_TL
+    )
+    ti_now = tlm_TL["TL.SH.TI"]
+
+    # 何も無いところに登録
+    assert "SUC" == wings.util.send_rt_cmd_and_confirm(
+        ope,
+        c2a_enum.Cmd_CODE_TEST_CCP_REGISTER_TLC_ASAP,
+        (ti_now + 10000,),
+        c2a_enum.Tlm_CODE_HK,
+    )
+
+    # 確認
+    tlm_TL = wings.util.generate_and_receive_tlm(
+        ope, c2a_enum.Cmd_CODE_GENERATE_TLM, c2a_enum.Tlm_CODE_TL
+    )
+    assert tlm_TL["TL.CMD0_TI"] == ti_now + 10000
+
+    # 同時刻に登録
+    assert "SUC" == wings.util.send_rt_cmd_and_confirm(
+        ope,
+        c2a_enum.Cmd_CODE_TEST_CCP_REGISTER_TLC_ASAP,
+        (ti_now + 10000,),
+        c2a_enum.Tlm_CODE_HK,
+    )
+
+    # 確認
+    tlm_TL = wings.util.generate_and_receive_tlm(
+        ope, c2a_enum.Cmd_CODE_GENERATE_TLM, c2a_enum.Tlm_CODE_TL
+    )
+    assert tlm_TL["TL.CMD0_TI"] == ti_now + 10000
+    assert tlm_TL["TL.CMD1_TI"] == ti_now + 10001
+
+    # 末尾に登録
+    assert "SUC" == wings.util.send_rt_cmd_and_confirm(
+        ope,
+        c2a_enum.Cmd_CODE_TEST_CCP_REGISTER_TLC_ASAP,
+        (ti_now + 10002,),
+        c2a_enum.Tlm_CODE_HK,
+    )
+
+    # 確認
+    tlm_TL = wings.util.generate_and_receive_tlm(
+        ope, c2a_enum.Cmd_CODE_GENERATE_TLM, c2a_enum.Tlm_CODE_TL
+    )
```

Comment:
[IMO]

```suggestion
    tlm_TL =get_latest_tl_tlm()
```
みたいに関数化したいかも（ `generate_and_receive_tlm` が３行にわたってばさばさはいってると，見通しよくないんだよね）

Reply:
これも修正しました